alfresco-docker-miscellaneous-transformers/src/main/java/org/alfresco/transformer/MiscellaneousTransformersController.java

```diff
@@ -0,0 +1,152 @@
+ * the paid license agreement will prevail.  Otherwise, the software is
+ * provided under the following open source license terms:
+ * -
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * -
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ * -
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+package org.alfresco.transformer;
+
+import org.alfresco.transform.client.model.Mimetype;
+import org.alfresco.transformer.transformers.SelectingTransformer;
+import org.alfresco.transformer.logging.LogEntry;
+import org.alfresco.transformer.probes.ProbeTestTransform;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.io.Resource;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.multipart.MultipartFile;
+
+import javax.servlet.http.HttpServletRequest;
+import java.io.File;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.alfresco.transformer.transformers.HtmlParserContentTransformer.SOURCE_ENCODING;
+import static org.alfresco.transformer.transformers.HtmlParserContentTransformer.TARGET_ENCODING;
+import static org.alfresco.transformer.fs.FileManager.createAttachment;
+import static org.alfresco.transformer.fs.FileManager.createSourceFile;
+import static org.alfresco.transformer.fs.FileManager.createTargetFile;
+import static org.alfresco.transformer.fs.FileManager.createTargetFileName;
+import static org.alfresco.transformer.logging.StandardMessages.ENTERPRISE_LICENCE;
+import static org.springframework.http.HttpStatus.OK;
+
+@Controller
+public class MiscellaneousTransformersController extends AbstractTransformerController
+{
+    private static final Logger logger = LoggerFactory.getLogger(MiscellaneousTransformersController.class);
+
+    @Autowired
+    private SelectingTransformer transformer;
+
+    public MiscellaneousTransformersController()
+    {
+        logger.info("--------------------------------------------------------------------------------------------------------------------------------------------------------------");
+        Arrays.stream(ENTERPRISE_LICENCE.split("\\n")).forEach(logger::info);

```

Comment:
Please double check this code has not changed, as the T-Engines have recently been opened up to be not enterprise only.

Reply:
Erik raised here: https://issues.alfresco.com/jira/browse/ATS-460